src/api.js

```diff
@@ -17,3 +17,13 @@ export const fetchTopics = () => {
       return data;
     });
 };
+
+export const fetchArticleByID = (article_id) => {
+  return axios
+    .get(`https://be-nc-news-je123.herokuapp.com/api/articles/${article_id}`, {
+      params: { article_id }, 
```

Comment:
`params` is unnecessary in this case because `article_id` is not a query option in this endpoint, and this api call will work just fine without this.